spec/models/agents/webhook_agent_spec.rb

```diff
@@ -59,6 +59,27 @@
       expect(out).to eq(['Event Created', 201])
     end
 
+    it 'should respond with customized response code if configured with `code` option' do
+      agent.options['code'] = '200'
+      out = agent.receive_web_request({ 'secret' => 'foobar', 'some_key' => payload }, "post", "text/html")
+      expect(out).to eq(['Event Created', 200])
+
+      # Empty string is a valid code
+      agent.options['code'] = ''
```

Comment:
Maybe move those tests to the second `it` block because it is also ensuring the previous defaults are kept?